internal/adapters/cloud/aws/ec2/vpc.go

```diff
@@ -139,17 +139,40 @@ func (a *adapter) adaptNetworkACL(apiNacl types.NetworkAcl) (*ec2.NetworkACL, er
 	return nacl, nil
 }
 
-func (a *adapter) adaptVPC(v types.Vpc) (*ec2.DefaultVPC, error) {
+func (a *adapter) adaptVPC(v types.Vpc) (*ec2.VPC, error) {
```

Comment:
`adaptVPC` now returns a `ec2.VPC` object rather than a `ec2.DefaultVPC` - there is a `IsDefault` value that shows it is what previously was known as `DefaultVPC`